Source/Lib/Codec/EbEncHandle.c

```diff
@@ -3282,7 +3282,7 @@ EB_ERRORTYPE EbH265EncInitParameter(
     configPtr->improveSharpness = EB_FALSE;
 
     // Bitstream options
-    configPtr->codeVpsSpsPps = 0;
+    configPtr->codeVpsSpsPps = 1;
     configPtr->codeEosNal    = 0;
```

Comment:
please verify ffmpeg plugin

Reply:
App enabled, not the library.

Reply:
Even though codeVpsSpsPps is set to 1 (as default) in encoder, it wouldn't have the side effect to insert redundant VPS/SPS/PPS. Becuase in SVT-HEVC ffmpeg plugin, codeVpsSpsPps will be reset to 0 when AV_CODEC_FLAG_GLOBAL_HEADER is set into the bitmap of AVCodec due to AVFMT_GLOBALHEADER is assigned for most containers (such as mp4 implemented by movenc).

So keep codeVpsSpsPps with 1 as default in encoder.

Reply:
BTW, keep codeVpsSpsPps with 1, in case that EbH265EncStreamHeader() wouldn't be invoked by any application to insert VPS/SPS/PPS.